src/midend/binaryAnalyses/instructionSemantics/DataFlowSemantics2.C

```diff
@@ -77,6 +77,7 @@ public:
     createOptionalMerge(const BaseSemantics::SValuePtr &other, const BaseSemantics::MergerPtr&,
                         SMTSolver*) const ROSE_OVERRIDE {
         TODO("[Robb P. Matzke 2015-08-10]");
```

Comment:
OK, this seems to compile:
  virtual __declspec(noreturn) Sawyer::Optional<BaseSemantics::SValuePtr>
So, it should be made into a macro.   ROSE_NORETURN?   Where should that macro go?